docs/resources/aws_api_gateway_stage.md

```diff
@@ -0,0 +1,112 @@
+---
+title: About the aws_api_gateway_stage Resource
+platform: aws
+---
+
+# aws\_api\_gateway\_stage
+
+Use the `aws_api_gateway_stage` InSpec audit resource to test properties of the singular resource of AWS ApiGateway Stage.
+
+The AWS::ApiGateway::Stage resource deploys an API Gateway RestApi resource to a stage so that clients can call the API over the internet. The stage acts as an environment.
```

Comment:
```suggestion
The `AWS::ApiGateway::Stage` resource deploys an API Gateway REST API resource to a stage so that clients can call the API over the internet. The stage acts as an environment.
```